src/app/analytics/analytics.factory.js

```diff
@@ -38,7 +38,8 @@ const analyticsFactory = /* @ngInject */ function ($window, $timeout, sessionSer
             item = {
               productInfo: {
                 productID: cartData.items[i].designationNumber,
-                designationType: cartData.items[i].designationType
+                designationType: cartData.items[i].designationType,
+                orgId: cartData.items[i].orgId ?? 'cru'
```

Comment:
Not sure if the ```?? cru``` is actually needed. From what I observed, every product had an orgId.